package/opt/hassbian/suites/cloud9.sh

```diff
@@ -29,7 +29,11 @@ sudo -u homeassistant -H /bin/bash << EOF
   printf "Downloading and installing Cloud9 SDK...\\n"
   git clone git://github.com/c9/core.git /opt/c9sdk
   bash /opt/c9sdk/scripts/install-sdk.sh
-  echo '{"projecttree": {"@showhidden": true,"@hiddenFilePattern": ".n*,*c9*,.b*,.p*,.w*,*.db"}}' | tee /home/homeassistant/.c9/user.settings
+  echo '{"projecttree": {"@showhidden": false,"@hiddenFilePattern": ".n*,*c9*,.b*,.p*,.w*,*.db"}}' | tee /home/homeassistant/.c9/user.settings
+  mkdir /home/homeassistant/c9workspace
+  ln -s /home/homeassistant/.homeassistant/ /home/homeassistant/c9workspace
+  ln -s /home/homeassistant/.c9 /home/homeassistant/c9workspace
+  mv /home/homeassistant/c9workspace/.homeassistant /home/homeassistant/c9workspace/homeassistant
```

Comment:
This line makes no sense to me?
 The symlink created should be c9workspace and shouldn't contain this directory?